package.json

```diff
@@ -93,14 +94,14 @@
   "scripts": {
     "build": "nps build && bundlesize",
     "watch": "nps build.watch",
-    "cover": "cross-env NODE_ENV=test nyc --reporter=lcov --reporter=text mocha",
-    "coveralls": "cross-env NODE_ENV=test nyc --reporter=lcov --reporter=text mocha && cat coverage/lcov.info | coveralls",
+    "coveralls": "cat coverage/lcov.info | coveralls",
     "lint": "eslint .",
     "deploy": "cross-env NODE_ENV=production nps publish",
     "start": "webpack-dev-server --progress",
     "fresh": "rm -rf node_modules && yarn install",
     "test": "npm run test:jest && npm run test:cypress",
     "test:jest": "jest --coverage",
+    "e2e": "concurrently --kill-others --success=first --names 'SERVER,E2E' 'yarn start' 'yarn test:cypress'",
```

Comment:
Could you change it to this command, please?

`"e2e": "concurrently --kill-others --success=first --names 'SERVER,E2E' 'yarn start > /dev/null 2>&1 'yarn test:cypress'",`

So that we can read the Cypress logs on Circle CI and remove the debug which is enabled on Webpack when it runs on develop mode

Reply:
Done 👍

Reply:
Sorry @JedWatson . That's because `concurrently` package requires output. I removed the output via `webpack-dev-server` and it's working fine. Could you update the command, please?

`"e2e": "concurrently --kill-others --success=first --names 'SERVER,E2E' 'yarn start --progress=false --no-info' 'yarn test:cypress'",`